lib/build/TaskRunner.js

```diff
@@ -309,12 +309,13 @@ class TaskRunner {
 					params.dependencies = dependencies;
 				}
 
-				if (task.getSpecVersion() === "3.0") {
+				const specVersion = task.getSpecVersionComparator();
```

Comment:
I struggle a bit with the naming of both the class and the getter. Here it's used with `specVersion` as local variable which reads fine (specVersion is greater then or equal to 3.0). But `specVersionComparator.gte("3.0")` would be strange.
But I also don't have a better idea, also as `getSpecVersion()` already returns the string. My only idea is to combine both to have `getSpecVersion().toString()` and `getSpecVersion.gte("3.0")`

Reply:
I'm not happy with the naming either.

I guess we could make this a "SpecVersion" class and have getSpecVersion() always return an instance of it. Indeed that might make things a bit simpler in the end.

Reply:
Yes, that sounds good 👍🏻

Reply:
Done in latest commit. I think this is way better, since the main use of a project's spec version is to compare it anyways 👍